### PR TITLE
docs: updated pull request template

### LINKED ISSUE #91: docs: Improve Pull request templates
### Is your feature request related to a problem? Please describe.
As I saw in the Issue template there are descriptions like `A clear and concise description of what you want to happen.` which is also shown in the preview window unless I remove it manually so it was a time taking process, the description like above should be commented as they are for Author's eye only.
Also, remove the irrelevant options like `I have written Kotlin Docs whenever is applicable`  under the checklist.

### Describe the solution you'd like
It should be commented like this `<!-- A clear and concise description of what you want to happen. -->`

### Describe alternatives you've considered
N/A

### Additional context
N/A

### PR DESCRIPTION
**Pull request by 17sushmita**
### Description

Fixes #91 

### Type of Change:

- Documentation

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -32,12 +32,13 @@ Fixes #ISSUE
 - [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
 - [ ] I have made corresponding changes to the documentation
 - [ ] Any dependent changes have been merged
-- [ ] I have written Kotlin Docs whenever is applicable
+- [ ] Update Postman API at /docs folder
```
Comment (codesankalp) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- [ ] I have updated Postman API inside [docs](https://github.com/anitab-org/open-source-programs-backend/blob/develop/docs/OSP.postman_collection.json)
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -32,12 +32,13 @@ Fixes #ISSUE
...
+- [ ] Update requirements.txt
```
Comment (codesankalp) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- [ ] I have updated dependencies in `requirements.txt`
```

**Review by codesankalp — COMMENTED** [review]
Thanks for your contribution @17sushmita.
I have mentioned some changes please look into it.

**Review by codesankalp — APPROVED** [review]
Thanks for your contribution @17sushmita.
:tada: :tada: 

I wonder the issue contains an instruction of commenting on some lines but I don't see any comments in changes.
@Amulya-coder Can you review it?

**Comment by 17sushmita** [comment]
> Thanks for your contribution @17sushmita.
> 🎉 🎉
> 
> I wonder the issue contains an instruction of commenting on some lines but I don't see any comments in changes.
> @Amulya-coder Can you review it?

@codesankalp , I guess the issue with uncommented instruction was with ISSUE_TEMPLATE which has now been deleted from the repository. However, it is still there in the ISSUE_TEMPLATE in `.github` repository. We can open an issue regarding this in that repository.

**Comment by Amulya-coder** [comment]
Yes @codesankaalp I saw now the Issue_template was deleted from the repository which contains uncommented comments that's why @isabelcosta changes the title of the issue to only improve pull request, as of now we merge this pr for the pull request improvements but we have to open another issue in .github repository as it's moved there.

**Review by Amulya-coder — APPROVED** [review]
Good work @17sushmita 🎉

**Pull request merged**